src/SFML/Window/OSX/HIDInputManager.mm

```diff
@@ -45,6 +45,7 @@
 ////////////////////////////////////////////////////////////
 bool HIDInputManager::isKeyPressed(Keyboard::Key key)
 {
+	if(key==Keyboard::CAPS)return false;
```

Comment:
This should be handled gracefully by `isPressed()`, e.g. by setting a special/undefined index for `m_keys[key]`. (Didn't check actual values/usage.)